tests/ssl_certs/chained.pem

```diff
@@ -0,0 +1,43 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can you updated the gencerts.sh file to show how this certificate was generated in case we need to reproduce a new one in the future.